compiler/include/ForallStmt.h

```diff
@@ -22,7 +22,47 @@
 
 #include "stmt.h"
 
-// forall loop statement
+
+//////////////////////////
+    // a forall intent //
+   //////////////////////////
```

Comment:
Let's put a TODO somewhere (in an issue? in a comment?) that we need to think about how to merge ForallIntent and TaskIntent so that the parser only needs to emit one of these, since they are syntactically the same (aren't they)?